### PR TITLE
Energy

### PR DESCRIPTION
**Pull request by yuanqing-wang**
implemented modularized normalizing mechanism

### CODE REVIEW

**Comment by yuanqing-wang** [comment]
@maxentile I added your MD samples code to the dataset

**Comment by yuanqing-wang** [comment]
I've added tests here to ensure the consistency between energy calculations of `espaloma` and `openmm`. @maxentile @jchodera

File: espaloma/__init__.py [hunk 1]
```diff
@@ -4,9 +4,18 @@
 """
 
 # Add imports here
-import espaloma
+# import espaloma
+from . import units, metrics
+
+#
+# from openforcefield.utils.toolkits import ToolkitRegistry, OpenEyeToolkitWrapper, RDKitToolkitWrapper, AmberToolsToolkitWrapper
```
Comment (maxentile) [inline espaloma/__init__.py:11]: Can this be removed?

File: espaloma/app/experiment.py [hunk 2]
```diff
@@ -101,6 +146,7 @@ def __init__(
         data,
         states,
         metrics=[esp.metrics.TypingCrossEntropy()],
+        normalize=esp.data.normalize.ESOL100LogNormalNormalize,
```
Comment (maxentile) [inline espaloma/app/experiment.py:149]: Could we add a comment here about what contexts will cause this normalization to be applied? I imagine this isn't being applied to the listed metric (cross-entropy loss), and only kicks in for other metrics (those for energy regression, parameter regression).

**Pull request merged**